### PR TITLE
Fix broken link and some typos

### PR DESCRIPTION
**Pull request by shubheksha**
- Fix broken link for meeting mins
- Add link to the milestone burndown google group which is to be joined for the calendar invite

### CODE REVIEW

File: releases/release-1.13/README.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ For documentation PRs:
 
 ### Burndown
 
-Burndown meetings are held on Mondays, Wednesdays and Fridays at TIME TBD until the final release is near, and then every business day until the release.  These meetings may overlap with other SIG meetings and the Community Meeting, but it is critical that SIGs provide representation at the release burndown meeting when requested to discuss specific issues and insure they do not become gating on the release. The release lead or a release team representative will keep the Community Meeting informed of release status even if there are meeting conflicts.
+Burndown meetings are held on Mondays, Wednesdays and Fridays at TIME TBD until the final release is near, and then every business day until the release.  These meetings may overlap with other SIG meetings and the Community Meeting, but it is critical that SIGs provide representation at the release burndown meeting when requested to discuss specific issues and ensure they do not become gating on the release. The release lead or a release team representative will keep the Community Meeting informed of release status even if there are meeting conflicts.
```
Comment (AishSundar) [inline releases/release-1.13/README.md]: Can you make this 10am PST

Comment (shubheksha) [reply releases/release-1.13/README.md]: Fixed, thanks for pointing it out!

**Comment by shubheksha** [comment]
/hold cancel
@AishSundar, could you add the lgtm again?

**Pull request merged**